test/Riok.Mapperly.IntegrationTests/_snapshots/MapperTest.SnapshotGeneratedSource.verified.cs

```diff
@@ -183,7 +184,8 @@ private Riok.Mapperly.IntegrationTests.Models.InheritanceSubObject MapToInherita
 
         private partial Riok.Mapperly.IntegrationTests.Dto.TestObjectDto MapToDtoInternal(Riok.Mapperly.IntegrationTests.Models.TestObject testObject)
         {
-            var target = new Riok.Mapperly.IntegrationTests.Dto.TestObjectDto(DirectInt(testObject.CtorValue), ctorValue2: DirectInt(testObject.CtorValue2));
+            var target = new Riok.Mapperly.IntegrationTests.Dto.TestObjectDto(DirectInt(testObject.CtorValue), ctorValue2: DirectInt(testObject.CtorValue2))
+            {IntInitOnlyValue = DirectInt(testObject.IntInitOnlyValue), };
```

Comment:
Do we have any control over the formatting? Looks kinda strange to me

Reply:
Currently we use the high level roslyn syntax apis with `NormalizeWhitespace()` which results in this formatting. However, I removed the trailing comma which improves it a little. If we wanted to control this, we would ned to remove the normalize whitespace call and add whitespaces manually everywhere which wouldn't be effective IMO.